Compiling Errors in HierarchicalSigmoidLayer

The compiling errors are as following:

```
/Users/baidu/Documents/codeRep/paddle-master/Paddle/paddle/gserver/layers/HierarchicalSigmoidLayer.cpp:167:9: error: address of function 'useGpu' will always evaluate to 'true' [-Werror,-Wpointer-bool-conversion]
    if (useGpu) {
    ~~  ^~~~~~
/Users/baidu/Documents/codeRep/paddle-master/Paddle/paddle/gserver/layers/HierarchicalSigmoidLayer.cpp:167:9: note: prefix with the address-of operator to silence this warning
    if (useGpu) {
        ^
        &
1 error generated.
make[2]: *** [paddle/gserver/CMakeFiles/paddle_gserver.dir/layers/HierarchicalSigmoidLayer.cpp.o] Error 1
```
It seems that `useGpu_` rather than `useGpu` should be used here.

--- a/paddle/gserver/layers/HierarchicalSigmoidLayer.cpp
+++ b/paddle/gserver/layers/HierarchicalSigmoidLayer.cpp
@@ -164,7 +164,7 @@ void HierarchicalSigmoidLayer::backward(const UpdateCallback& callback) {
       cpuBias_ = biases_grad;
     }
     preOutput_.grad->addByBitCodeBackward(numClasses_, *cpuLabel_, *cpuBias_);
-    if (useGpu) {
+    if (useGpu_) {
       biases_grad->copyFrom(*cpuBias_);
     } else {
       biases_grad = cpuBias_;

